client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/OutboundLineEdit.tsx

```diff
@@ -32,28 +32,44 @@ import {
   getAllocatedQuantity,
   getAllocatedPacks,
 } from './utils';
-import { useOutbound } from '../../api';
+import { DraftItem, useOutbound } from '../../api';
 import { DraftOutboundLine } from '../../../types';
 
 interface ItemDetailsModalProps {
   isOpen: boolean;
   onClose: () => void;
-  item: ItemRowFragment | null;
+  draft: DraftItem | null;
   mode: ModalMode | null;
 }
 
+const useFocusNumberOfPacksInput = (draft: DraftItem | null) => {
+  const batch = draft?.barcode?.batch;
+
+  useEffect(() => {
+    setTimeout(() => {
+      if (!batch) return;
+      const input = document.getElementById(`pack_quantity_${batch}`);
```

Comment:
kind of like helper methods for constructing these types of string with constants in them (since it's used in at least two places)

Reply:
Added helper in 95790d660